docs/999-big-list-of-options.md

```diff
@@ -35,7 +35,7 @@ When given as a command line argument, it should be a comma-separated list of ID
 rollup -i src/main.js ... -e foo,bar,baz
 ```
 
-When providing a function, it is actually called with three parameters `(id, parent, isResolved)` that can give you more fine-grained control:
+When providing a function, it's called with three parameters `(id, parent, isResolved)` that can give you more fine-grained control:
```

Comment:
```suggestion
When providing a function, it is called with three parameters `(id, parent, isResolved)` that can give you more fine-grained control:
```

While there are some exceptions, we usually do not use contractions in the documentation (admittedly 9x "it's" vs. 51 times "it is")